borg/archiver.py

```diff
@@ -1812,6 +1813,7 @@ def build_parser(self, args=None, prog=None):
                                    type=CompressionSpec, default=None, metavar='COMPRESSION',
                                    help='select compression algorithm (and level):\n'
                                         'none == no compression (default),\n'
+                                        'auto,C[,L] == built-in heuristic decides between none or C[,L],\n'
```

Comment:
I have only this tiny nit pick: `auto,C[,L]` doesn't explain what C and L is. Maybe just say that C[,L] any other valid compression algo + optional level?